packages/create-emotion-styled/src/utils.js

```diff
@@ -7,14 +7,14 @@ export function setTheme(theme: Object) {
   this.setState({ theme })
 }
 
-export const testOmitPropsOnStringTag: (key: string) => boolean = memoize(
+export const testPickPropsOnStringTag: (key: string) => boolean = memoize(
   isPropValid
 )
-export const testOmitPropsOnComponent = (key: string) =>
+export const testPickPropsOnComponent = (key: string) =>
   key !== 'theme' && key !== 'innerRef'
 export const testAlwaysTrue = () => true
 
-export const omitAssign: (
+export const pickAssign: (
```

Comment:
imho `omit` was misleading here, and `pick` is better suited because `pickAssign` actually picks properties if predicate (`testFn`) returns truthy, `omit` would pick them if predicate would return falsy